src/patternfly/components/Chip/chip.scss

```diff
@@ -15,7 +15,7 @@
   // chip text
   --pf-c-chip__text--FontSize: var(--pf-global--FontSize--xs);
   --pf-c-chip__text--Color: var(--pf-global--Color--100);
-  --pf-c-chip__text--MaxWidth: #{pf-size-prem(120px)};
+  --pf-c-chip__text--MaxWidth: 16ch;
```

Comment:
nice! wdyt about making this update for the chip group label text, too?

https://github.com/patternfly/patternfly/blob/376703bdfbaf9ddc70f7d08e37f695079dba1c38/src/patternfly/components/ChipGroup/chip-group.scss#L19